packages/blade/src/hooks/useId.ts

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable @typescript-eslint/prefer-nullish-coalescing */
```

Comment:
put all the hooks and everything under utils directly?

Reply:
Not sure about that. 

Hooks are utils. they are hooks. We should have some distinction.